src/main/resources/assets/computercraft/lua/rom/programs/shell.lua

```diff
@@ -440,6 +441,17 @@ function shell.aliases()
     return tCopy
 end
 
+function shell.setPackagePath( sPath )
+    if type( sPath ) ~= "string" then
+        error( "bad argument #1 (expected string, got " .. type( sPath ) .. ")", 2 )
+    end
+    sPackagePath = sPath
+end
+
```

Comment:
I think that's the point, though I don't think it's a massive issue as long as it is documented. That being said, it might be a little clearer if it was get/setDefaultPackagePath, though that is a little verbose.